docs/resources/private_endpoint_connection.md

```diff
@@ -3,19 +3,82 @@
 page_title: "cockroach_private_endpoint_connection Resource - terraform-provider-cockroach"
 subcategory: ""
 description: |-
-  Private Endpoint Connection.
+  Private endpoint connections allow customer applications to connect to a CockroachDB Cloud cluster without traversing the public internet. All application-database traffic remains within the cloud-provider network.
 ---
 
 # cockroach_private_endpoint_connection (Resource)
 
-Private Endpoint Connection.
+Private endpoint connections allow customer applications to connect to a CockroachDB Cloud cluster without traversing the public internet. All application-database traffic remains within the cloud-provider network.
 
 ## Example Usage
 
 ```terraform
-resource "cockroach_private_endpoint_connection" "serverless" {
-  cluster_id  = cockroach_cluster.serverless.id
-  endpoint_id = "endpoint-id"
+## Example with AWS PrivateLink
+
+# Enable the PrivateLink service on the CockroachDB Cloud cluster.
+resource "cockroach_private_endpoint_services" "services" {
+  cluster_id = cockroach_cluster.my_cluster.id
+}
+
+# Create a PrivateLink endpoint and associate it with the PrivateLink Service. 
+resource "aws_vpc_endpoint" "my_endpoint" {
+  vpc_id             = "vpc-7fc0a543"
+  service_name       = cockroach_private_endpoint_services.services.name
+  vpc_endpoint_type  = "Interface"
+  subnet_ids         = ["subnet-de0406d2"]
+  security_group_ids = ["sg-3f238186"]
+}
+
+# Establish the connection between the endpoint and the service.
+resource "cockroach_private_endpoint_connection" "connection" {
+  cluster_id  = cockroach_cluster.my_cluster.id
+  endpoint_id = aws_vpc_endpoint.my_endpoint.id
+}
+
+## Example with Azure Private Link
+
+# Enable the Private Link service on the CockroachDB Cloud cluster.
+resource "cockroach_private_endpoint_services" "services" {
+  cluster_id = cockroach_cluster.my_cluster.id
+}
+
+# Create a private link endpoint and associate it with the Private Link Service. 
+resource "azurerm_private_endpoint" "my_endpoint" {
```

Comment:
is this right? `azurerm_`...?

Reply:
Yiup https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/private_endpoint

Reply:
oh, got it.  It's not ours.  👍